### DISCUSSION TITLE
Monorepo and "Team access required to deploy" checks

### DISCUSSION THREAD

**Discussion by PaulMonnery** [vercel/vercel#6234] [Help]
Hi everyone,

my team is working on a monorepo, with 2 front-end projects, and 2 back-end projects, it looks like this:
```
monorepo
├── dashboard_back
├── dashboard_front
├── lab_back
└── lab_front
```
I am using the monorepo feature to only create preview when the `lab_front` and `dashboard_front` folders are modified (I put `git diff --quiet HEAD^ HEAD ./` in the *Ignored Build Step* section in Git settings).

This works great when I'm coding on the front-end folders, but if a backend dev creates a PR while working on back-end folders it will tell him **Team access required to deploy** and make the check fail. Backend dev are not part of the vercel team because it's not related to their work at all.

Is there a way to check for *Ignored Build Step* **before** the team acces check?  
I don't want to add half of our dev team to the vercel team if they never work on front-end folders just to have CI checks passing...

Thank you

**Comment by yoshiko-pg** [comment]
Same here. We are using monorepo as well.
We recently signed up for the Preview Deployment Suffix for the Preview Deploy of the frontend directory. And I was trying to add the whole frontend team to the team by turning on GitHub integration, but I am about to give that all up when I found out that there is no way to make Ignored Build Step work before team access check.

Non-front-end team members don't need Vercel's features, including deploy.

**A way to make Ignored Build Step work before team access check** or **a way for GitHub Vercel bot to ignore changes outside the configured Root Directory** is a must for monorepo users. I am eagerly waiting for this feature.
